Mathlib/Analysis/SpecialFunctions/Log/ENNRealLog.lean

```diff
@@ -0,0 +1,228 @@
+/-
+Copyright (c) 2024 Damien Thomine. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Damien Thomine, Pietro Monticone
+-/
+import Mathlib.Analysis.SpecialFunctions.Pow.NNReal
+import Mathlib.Topology.Instances.EReal
+
+/-!
+# Extended nonnegative real logarithm
+We define `log` as an extension of the logarithm of a positive real
+to the extended nonnegative reals `ℝ≥0∞`. The function takes values
+in the extended reals `EReal`, with `log 0 = ⊥` and `log ⊤ = ⊤`.
+
+## Main definitions
+- `log`: The extension of the real logarithm to `ℝ≥0∞`.
```

Comment:
Can you mention the namespaces in the references to statements in docstrings? for instance, here it should be
```suggestion
- `ENNReal.log`: The extension of the real logarithm to `ℝ≥0∞`.
```